dll/win32/shell32/lang/fr-FR.rc

```diff
@@ -48,11 +48,11 @@ BEGIN
     MENUITEM SEPARATOR
     MENUITEM "C&ouper\tCtrl+X", FCIDM_SHVIEW_CUT
     MENUITEM "&Copier\tCtrl+C", FCIDM_SHVIEW_COPY
-    MENUITEM "Co&ller\tCtrl+V", FCIDM_SHVIEW_INSERT
-    MENUITEM "Créer un &lien", FCIDM_SHVIEW_INSERTLINK
+    MENUITEM "Co&ller\tCtrl+V", FCIDM_SHVIEW_INSERT //FIXME: accelerator collision &L
+    MENUITEM "Créer un &lien", FCIDM_SHVIEW_INSERTLINK //FIXME: accelerator collision &L
     MENUITEM SEPARATOR
-    MENUITEM "Copier vers le &dossier...", FCIDM_SHVIEW_COPYTO
-    MENUITEM "&Déplacer vers le dossier...", FCIDM_SHVIEW_MOVETO
+    MENUITEM "Copier vers le &dossier...", FCIDM_SHVIEW_COPYTO //FIXME: accelerator collision &D
+    MENUITEM "&Déplacer vers le dossier...", FCIDM_SHVIEW_MOVETO //FIXME: accelerator collision &D
```

Comment:
```suggestion
    MENUITEM "Co&pier vers le &dossier...", FCIDM_SHVIEW_COPYTO
    MENUITEM "&Déplacer vers le dossier...", FCIDM_SHVIEW_MOVETO
```
it looks like the "p" accelerator wasn't used so far in this menu, and thus could be used there.

Reply:
Thank you very much for your accelerator suggestions as a native french speaker with a french Windows at hand. Finally a useful review-comment. I applied all your suggestions in the accelerator context.